home-excercises-05-2022/src/main/java/io/codelex/loops/practice/RollTwoDice.java

```diff
@@ -0,0 +1,23 @@
+package io.codelex.loops.practice;
+
+import java.util.Random;
+import java.util.Scanner;
+
+
+public class RollTwoDice {
+    public static void main(String[] args) {
+        System.out.print("Please enter Your desired sum: ");
+        Scanner input = new Scanner(System.in);
+        int desired = input.nextInt();
+        Random gen = new Random();
+        int firstNum;
+        int secNum;
+        int sum;
+        do {
+            firstNum = gen.nextInt(6) + 1;
+            secNum = gen.nextInt(6) + 1;
```

Comment:
arī kā divreiz atkāŗtojas, potenciāls iznest uz privāto `rollDice()`, daudz lasāmāk

Reply:
+